src/bot/discord_bot.py

```diff
@@ -423,18 +424,64 @@ async def _add_user(m, options):
     },
 }
 
-# BEGIN CLIENT EVENTS -----
+# SLASH COMMANDS
 
+tree = app_commands.CommandTree(client)
 
 @client.event
 async def on_ready() -> None:
     await on_startup()
+    await tree.sync(guild=discord.Object(id=1080421430426091551))
+
+# MOVIE SLASH COMMAND
+@tree.command(name="movie", description="Request a movie", guild=discord.Object(id=1080421430426091551))
```

Comment:
I assume this guild-id is a placeholder for debugging, right?

Reply:
Yes, it's just for debugging. Thought I removed that, my bad!